customer-churn/streamlit_apps/requirements.txt

```diff
@@ -0,0 +1 @@
+zenml==0.7.3
```

Comment:
Once we're out with 0.8.0, this should be upgraded. So if we release the zenfile this week, then next week you should update and test it with 0.8.0.

Reply:
I have tested it on develop branch, I will update it once 0.8.0 is launched.